components/export_table/ExportTable.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+
+const teacher = [
+  {
+    id: 1,
```

Comment:
extract teacher object into typescript type

create a `teacher.ts` in `model/`
don't forget to `export` the type and type the array to `Teacher[]`